packages/config/src/eslint.cjs

```diff
@@ -63,7 +66,15 @@ function createEslintConfig(
       'src/types/auto-generated.d.ts',
     ],
     plugins: ['vue', 'import', '@typescript-eslint'],
-    extends: ['eslint:recommended', 'plugin:vue/vue3-recommended', 'standard', '@vue/typescript', 'plugin:import/recommended', 'plugin:import/typescript'],
+    extends: [
+      'eslint:recommended',
+      'plugin:vue/vue3-recommended',
+      'standard',
+      '@vue/typescript',
+      'plugin:import/recommended',
+      'plugin:import/typescript',
+      'plugin:json-schema-validator/recommended'
+    ],
```

Comment:
We have a shared ESLint v9+ config for the frontend ecosystem if you'd like to adopt it: https://github.com/Kong/eslint-config-kong-ui

Reply:
At the moment I think that only supports eslint 9 (correct me if I'm wrong!) we are still on 8. We have an issue to upgrade it tho - https://github.com/kumahq/kuma-gui/issues/2879